src/ansys/units/map.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, item):
 
 class QuantityMap(object):
     """
-    A class that contains quantity map and equivalent units.
+    A class that represents quantity map and equivalent units.
```

Comment:
Now that we know what this is we could say `A class that represents a mapping between a named quantity and it's units`.
```suggestion
    A class that represents a mapping between a named quantity and it's equivalent units.
```